README.md

```diff
@@ -45,15 +45,15 @@ The Okteto CLI has two operational modes:
 
 ### Okteto Open Source CLI Features
 
-Okteto Open Source requires access to a Kubernetes cluster. It's designed to support Development Containers in any Kubernetes cluster. It doesn't come with features that support multiple developers working on the same cluster. That's the goal of the [Okteto Platform](https://okteto.com/docs)
+The Okteto Open Source CLI requires access to a Kubernetes cluster. The Okteto Open Source CLI aims to enable you to develop your cloud-native applications using Development Containers in your Kubernetes clusters. If you are interested in helping your entire team share a Kubernetes cluster for development, we recommend you check out our commercial product, the [Okteto Platform](https://okteto.com)
 
-Okteto Open Source supports the following commands:
+The Okteto Open Source CLI supports the following commands:
 
 - `okteto context`
 - `okteto up`
 - `okteto down`
 
-For reference, our Open Source CLI supports the [dev section](https://www.okteto.com/docs/reference/okteto-manifest/#dev-object-optional) of the Okteto Manifest.
+> It's important to note that the Okteto Open Source CLI only supports the [dev section](https://www.okteto.com/docs/reference/okteto-manifest/#dev-object-optional) of the Okteto Manifest.
```

Comment:
@codyjlandstrom I find it confusing that we are sending people that are using the Okteto CLI in pure cluster mode to https://okteto.com/docs for reference. Do we have an in-repo description of the manifest? would that make sense?

Reply:
Maybe we could have a simplified one? I fear that the full dev section of the manifest is too long to place here and keep maintained

Reply:
yeah, that's what I was thinking as well. it can help reinforce the difference. but that's out o the scope of this PR.